net: lib: nrf_cloud: Clean up KConfigs so they don't clutter unrelated samples (#20074)
nrfconnect/sdk-nrf: doc/nrf/libraries/networking/nrf_cloud_agnss.rst

Configuration
*************

> Enable :kconfig:option:`CONFIG_NRF_CLOUD` and configure the following Kconfig options to enable or disable the use of this library:

Reviewer: ```suggestion
To enable this library, set the :kconfig:option:`CONFIG_NRF_CLOUD` Kconfig option and configure the following options:
```
